schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/InterpretedGenomesRD.avdl

```diff
@@ -64,7 +52,7 @@ protocol InterpretedGenomesRD {
         /**
         List of all reported structural variants
         */
-        union {null, array<ReportedStructuralVariant>} reportedStructuralVariants;
+        union {null, array<ReportedStructuralVariant>} structuralVariants;
 
         /**
         Comments about the report
```

Comment:
we need to think about the two possibilities of interaction:
InterpretationRequest->InterpretationService->InterpretedGenome->MDTDelivery->...
or just
MDTDelivery (with interpretation request)...